oauthdoctor/bin/gen-bin.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+root=<Enter your Google Ads Doctor root path here>
+bin=$root/bin
+
+# Remove previously generated binaries
+rm -f $root/windows
+rm -f $root/osx
+rm -f $root/linux
+
+# Generate binaries from master branch
+pushd $root
+orig_branch=$(git rev-parse --abbrev-ref HEAD)
+git checkout master
```

Comment:
cur_branch=$(git branch | grep \* | cut -d ' ' -f2)
if [[ "${cur_branch}" != "master" ]] ; then
  git checkout master
 fi

Reply:
I stick to my git command because that's more straight forward to a git user